src/state/queries/feed.ts

```diff
@@ -304,31 +314,24 @@ export function useGetPopularFeedsQuery(options?: GetPopularFeedsOptions) {
 
 export function useSearchPopularFeedsMutation() {
   const agent = useAgent()
+  const moderationOpts = useModerationOpts()
+
   return useMutation({
     mutationFn: async (query: string) => {
       const res = await agent.app.bsky.unspecced.getPopularFeedGenerators({
         limit: 10,
         query: query,
       })
 
-      return res.data.feeds
-    },
-  })
-}
-
-export function useSearchPopularFeedsQuery({q}: {q: string}) {
-  const agent = useAgent()
-  return useQuery({
-    queryKey: ['searchPopularFeeds', q],
-    queryFn: async () => {
-      const res = await agent.app.bsky.unspecced.getPopularFeedGenerators({
-        limit: 15,
-        query: q,
-      })
+      if (moderationOpts) {
+        return res.data.feeds.filter(feed => {
+          const decision = moderateFeedGenerator(feed, moderationOpts)
+          return !decision.ui('contentList').filter
+        })
+      }
```

Comment:
Mutations don't support `enabled`, but this method is called after load when the user types in the search input on Feeds screen. So even though we have a check, it should be present.

I spent a little time trying to swap for the other search query in here (not a mutation) but decided it wasn't worth the effort right now to refactor the Feeds screen to use it. This was a quick simple approach that works.